lib/matrix/raw_client.ex

```diff
@@ -42,7 +42,7 @@ defmodule M51.Matrix.RawClient do
         {:error, status_code, body}
```

Comment:
You can remove that; it was only meant to skip the JSON-decoding you just removed

Reply:
Oh, right, guess you meant this case:
```diff
-      {:ok, %HTTPoison.Response{status_code: status_code, body: <<?<, _::binary>> = body}} ->
-        # That's not JSON!
-        {:error, status_code, body}
```